.vib/fluentd/goss/goss.yaml

```diff
@@ -0,0 +1,31 @@
+
+file:
+  /opt/bitnami/fluentd/Gemfile:
+    mode: "0644"
+    filetype: file
+    exists: true  
+command:
+  fluentd-conf-test:
+    exec: fluentd -c /opt/bitnami/fluentd/conf/fluentd.conf --dry-run
+    exit-status: 0
+    stdout: 
+      - {{ printf "/bitnami.*%s/" .Env.BITNAMI_APP_NAME }}
+      - {{ printf "/port *%s/" .Env.FLUENTD_AGGREGATOR_SERVICE_PORT }}
+      - {{ printf "/port *%s/" .Env.FLUENTD_AGGREGATOR_SERVICE_PORT_TCP }}
```

Comment:
After briefly checking the chart's design, I don't it makes sense to templatize this (or maybe even check)

The `fluentd.conf` file is not generated dynamically, but rather set to a fixed value:

https://github.com/bitnami/charts/blob/819db25f92b96b0a883cdae95228a961e4d83e35/bitnami/fluentd/values.yaml#L166-L170

Hence, even if the chart is configured with another service_port, the file won't be updated automatically.

Reply:
OK, then it makes more sense to have this test check only the presence of the config file.